skillserver/skillserver.go

```diff
@@ -58,12 +59,26 @@ func SetRootPrefix(prefix string) {
 	RootPrefix = prefix
 }
 
-func RunSSL(apps map[string]interface{}, port, cert, key string) error {
-	router := mux.NewRouter()
-	Init(apps, router)
-
-	err := http.ListenAndServeTLS(port, cert, key, router)
-	return err
+func RunSSL(apps map[string]interface{}, port, cert, key string) {
```

Comment:
Would be good to doc this function with a proper [doc comment](https://blog.golang.org/godoc-documenting-go-code)